bin/index_db.sh

```diff
@@ -166,6 +166,13 @@ sed -i s#%PIG_PATH#$BASE_WDIR/pig/pignlproc#g examples/indexing/names_and_entiti
 sed -i s#/user/hadoop#/user/$USER#g examples/indexing/token_counts.pig.params
 sed -i s#/user/hadoop#/user/$USER#g examples/indexing/names_and_entities.pig.params
 
+#Add macros
+echo "MACROS_DIR=$1/pig/pignlproc/examples/macros/" >> examples/indexing/token_counts.pig.params
```

Comment:
I think these lines have been added in the pignlproc repo itself, so you might not need them here.